packages/alfa-rules/src/sia-r113/rule.ts

```diff
@@ -0,0 +1,145 @@
+import { Rule } from "@siteimprove/alfa-act";
+import { Cache } from "@siteimprove/alfa-cache";
+import { Device } from "@siteimprove/alfa-device";
+import { Document, Element } from "@siteimprove/alfa-dom";
+import { Predicate } from "@siteimprove/alfa-predicate";
+import { Rectangle } from "@siteimprove/alfa-rectangle";
+import { Err, Ok } from "@siteimprove/alfa-result";
+import { Criterion } from "@siteimprove/alfa-wcag";
+import { Page } from "@siteimprove/alfa-web";
+
+import { expectation } from "../common/act/expectation";
+
+import { targetsOfPointerEvents } from "../common/applicability/targets-of-pointer-events";
+
+import { WithBoundingBox, WithName } from "../common/diagnostic";
+
+import { hasSufficientSize } from "../common/predicate/has-sufficient-size";
+import { isUserAgentControlled } from "../common/predicate/is-user-agent-controlled";
+
+const { or } = Predicate;
+
+export default Rule.Atomic.of<Page, Element>({
+  uri: "https://alfa.siteimprove.com/rules/sia-r113",
+  requirements: [Criterion.of("2.5.8")],
+  evaluate({ device, document }) {
+    return {
+      applicability() {
+        return targetsOfPointerEvents(document, device);
+      },
+
+      expectations(target) {
+        // Existence of a bounding box is guaranteed by applicability
+        const box = target.getBoundingBox(device).getUnsafe();
+        const name = WithName.getName(target, device).getOr("");
+        return {
+          1: expectation(
+            isUserAgentControlled()(target),
+            () => Outcomes.IsUserAgentControlled(name, box),
+            hasSufficientSizeOrSpacing(document, device)(target)
+              ? () => Outcomes.HasSufficientSizeOrSpacing(name, box)
+              : () => Outcomes.HasInsufficientSizeAndSpacing(name, box),
```

Comment:
```suggestion
            expectation(hasSufficientSizeOrSpacing(document, device)(target),
              () => Outcomes.HasSufficientSizeOrSpacing(name, box),
              () => Outcomes.HasInsufficientSizeAndSpacing(name, box))
```
I think we should also split the test further (size first then spacing) which would make different outcomes for both cases.